src/copyright/ui/ajax-copyright-hist.php

```diff
@@ -12,6 +12,7 @@
 use Fossology\Lib\Db\DbManager;
 use Fossology\Lib\Util\DataTablesUtility;
 use Fossology\Agent\Copyright\UI\TextFindingsAjax;
+use PhpOffice\PhpSpreadsheet\Calculation\LookupRef\Offset;
```

Comment:
Why this class is needed in this page?

Reply:
This was an unused class..... removing it